pkg/skaffold/instrumentation/meter.go

```diff
@@ -97,6 +97,10 @@ func init() {
 	meteredCommands.Insert("build", "delete", "deploy", "dev", "debug", "filter", "generate_pipeline", "render", "run", "test")
 	doesBuild.Insert("build", "render", "dev", "debug", "run")
 	doesDeploy.Insert("deploy", "dev", "debug", "run")
+}
+
+// SetOnlineStatus issues a GET request to see if the user is online
+func SetOnlineStatus() {
```

Comment:
I suggest linking to https://www.chromium.org/chromium-os/chromiumos-design-docs/network-portal-detection to indicate that this is not some internal tracking server.

Reply:
done 👍🏼